pytest.ini

```diff
@@ -26,4 +26,6 @@ addopts = --verbose
     --ignore=cylc/flow/parsec/empysupport.py
     --ignore=cylc/flow/parsec/example
 testpaths =
+    cylc/flow/
     tests/unit/
+    tests/integration/
```

Comment:
This allows VSCode's test explorer to pick up on those two folders

Reply:
This was purposfully omitted so that the default `pytest` just ran the unit tests as the integration test battery will become slower as it grows.

Personally I'm happy to change (can just do `pytest tests/u` if you only want unit tests).